_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+
+The Powder River Basin is composed of surface mines with giant coal seams as shown below.  This region produces around 42% of all coal in the US[^PowderRiverBasin_42_percent], with the North Antelope Rochelle Mine alone providing 12% of total US production in 2016[^NorthAntelope_12_percent].  This single mine produces more coal than West Virginia, the second largest coal mining state.
+
+![Coal Seam]({{ site.baseurl }}public/images/Coal_Seam_at_Coal_Mine.png)
+<sup>_Coal seam, [North Antelope Rochelle Mine](https://en.wikipedia.org/wiki/North_Antelope_Rochelle_Mine){:target="_blank"}. By Peabody Energy, Inc. [CC BY 3.0](https://creativecommons.org/licenses/by/3.0){:target="_blank"}, via [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Coal_Seam_at_Coal_Mine.png){:target="_blank"}_</sup>
+
+The animated map below by the [Google Earth Engine Timelapse](https://earthengine.google.com/timelapse/#v=43.60679,-105.24646,9.613,latLng&t=2.96){:target="_blank"} shows the enormous geographic scale of the Power River Basin mines, along with their historical growth over 32 years of satellite imagery.  Over time, one can see new areas being dug out with land restoration efforts following shortly behind.
+
+{% raw %}
+<div>
+<iframe style="position: absolute; top: -9999em; visibility: hidden;" onload="this.style.position='static'; this.style.visibility='visible';" src="https://earthengine.google.com/iframes/timelapse_player_embed.html#v=43.60679,-105.24646,9.826,latLng&t=1.82" width="700" height="500"></iframe>
+</div>
+{% endraw %}
+<sup>_Coal Mines, Powder River Basin, Wyoming, [Google Earth Engine Timelapse](https://earthengine.google.com/timelapse/#v=43.60679,-105.24646,9.613,latLng&t=2.96){:target="_blank"}_</sup>
+
+## Sulfur Content
+
+The EIA data on coal shipments has incredible resolution, and one can find information about quarterly shipments between individual mines and power plants.  For each of these there is information about the type of coal, ash content, heat content, price, quantity, and sulfur content.
+
+The sulfur content of coal is a concern due to [SO<sub>2</sub> pollution](https://www.epa.gov/so2-pollution/sulfur-dioxide-basics#effects){:target="_blank"} resulting from coal combustion, which can lead to problems such as acid rain, respiratory problems, and atmospheric haze.  While high sulfur content does not necessarily translate into high SO<sub>2</sub> emissions due to [desulfurization technology](https://en.wikipedia.org/wiki/Flue-gas_desulfurization){:target="_blank"} used by power plants to reduce emissions[^eGRID_Sulfur], the process of desulfurization is an economic cost rather than a benefit, and examining sulfur content can at a minimum give indications related to the [economics of coal prices](https://www.eia.gov/energyexplained/index.php?page=coal_prices){:target="_blank"}.
```

Comment:
--> "The sulfur content of coal is a concern due to [SO<sub>2</sub> pollution](https://www.epa.gov/so2-pollution/sulfur-dioxide-basics#effects){:target="_blank"} resulting from coal combustion, which can lead to acid rain, respiratory problems, and atmospheric haze. "